src/components/side_panel/components/radio_selection/radio_selection.ts

```diff
@@ -48,12 +49,11 @@ export class RadioSelection extends Component<Props, SpreadsheetChildEnv> {
   static props = {
     choices: Array,
     onChange: Function,
-    selectedValue: String,
+    selectedValue: { optional: false },
```

Comment:
`type: String`

Reply:
I want a type `any` (could be boolean, number, string or null) :x